### DISCUSSION TITLE
I can't publish a page if the html in the htlmbody contains required fields

### DISCUSSION THREAD

**Discussion by ghost** [OrchardCMS/OrchardCore#17149] [General]
I have a page that uses a simply contact form that I plan to connect to a work flow.   The issue I have now is that the form has required fields, but the required fields inside the form are preventing me from publishing the page.  Whenever I click publish, it focuses on my form and alerts for required input.

![image](https://github.com/user-attachments/assets/98775763-8e5e-4789-817d-7bfb9b6ce159)

**Comment by hishamco** [comment]
Can you share a minimal repo so we can reproduce the issue quickly

**Comment by ns8482e** [comment]
You have added HTML widget and created fields using html tags inside html widget?

Use form widgets instead to connect it to workflow.
